src/Proto.Mailbox/Mailbox.cs

```diff
@@ -5,11 +5,17 @@
 // -----------------------------------------------------------------------
 
 using System;
+using System.Runtime.CompilerServices;
 using System.Threading;
 using System.Threading.Tasks;
 
 namespace Proto.Mailbox
 {
+    public static class Tasks
```

Comment:
I was trying to share the Task object like `Actor.Done` but I guess having it internal is fine to not expose more code.